tests/ts/background/services/account-balance.test.ts

```diff
@@ -0,0 +1,23 @@
+import { getBalance } from '../../../../src/ts/background/services/account-balance'
+import { Alexander } from '../../../../src/ts/constants/networks'
+
+const ALICE = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY'
+const EMPTY_ACCOUNT = '5F5BvRY9MJc74UUc8tLV9XAvitPNNeEWZ6TVfXwH7cpf8mox'
+
+describe('Testing account-balance', () => {
+
+  test('getBalance', async () => {
+    const balance = await getBalance(ALICE, Alexander)
+    expect(balance).toBeGreaterThanOrEqual(0)
+  })
+
+  test('getBalanceWithFormat', async () => {
```

Comment:
if we really want to test this part, we probably can check if `formatBalance` has been invoked with an expect balance as the input.

Reply:
Removed this test, as it is unncessary